modules/Bio/EnsEMBL/Analysis/Hive/Config/production_rnaseq_db.pm

```diff
@@ -0,0 +1,590 @@
+    'production_db_server'         => 'mysql-ens-meta-prod-1',
+    'production_db_port'           => '4483',
+
+    databases_to_delete => ['rnaseq_db'],
+
+    ########################
+    # BLAST db paths
+    ########################
+    'base_blast_db_path'        => $ENV{BLASTDB_DIR},
+
+    ######################################################
+    #
+    # Mostly constant settings
+    #
+    ######################################################
+    ensembl_analysis_script           => catdir($self->o('enscode_root_dir'), 'ensembl-analysis', 'scripts'),
+    load_optimise_script              => catfile($self->o('ensembl_analysis_script'), 'genebuild', 'load_external_db_ids_and_optimize_af.pl'),
+
+    ensembl_misc_script        => catdir($self->o('enscode_root_dir'), 'ensembl', 'misc-scripts'),
+    meta_coord_script          => catfile($self->o('ensembl_misc_script'), 'meta_coord', 'update_meta_coord.pl'),
+    meta_levels_script         => catfile($self->o('ensembl_misc_script'), 'meta_levels.pl'),
+    assembly_name_script       => catfile($self->o('ensembl_analysis_script'), 'update_assembly_name.pl'),
+
+    rnaseq_daf_introns_file => catfile($self->o('output_dir'), 'rnaseq_daf_introns.dat'),
+
+    ########################
+    # Extra db settings
+    ########################
+    'num_tokens' => 10,
+
+    ########################
+    # Executable paths
+    ########################
+    deeptools_bamcoverage_path => '/nfs/software/ensembl/RHEL7-JUL2017-core2/pyenv/versions/genebuild/bin/bamCoverage',
+
+    'output_dir'    => catdir($self->o('rnaseq_dir'),'output'),
+    'merge_dir'     => catdir($self->o('rnaseq_dir'),'merge'),
+
+    use_threads => 3,
+    rnaseq_merge_threads => 12,
+    rnaseq_merge_type => 'samtools',
+    read_min_paired => 50,
+    read_min_mapped => 50,
+    other_isoforms => 'other', # If you don't want isoforms, set this to undef
+    maxintron => 200000,
+
+    # Please assign some or all columns from the summary file to the
+    # some or all of the following categories.  Multiple values can be
+    # separted with commas. ID, SM, DS, CN, is_paired, filename, read_length, is_13plus,
+    # is_mate_1 are required. If pairing_regex can work for you, set is_mate_1 to -1.
+    # You can use any other tag specified in the SAM specification:
+    # http://samtools.github.io/hts-specs/SAMv1.pdf
+
+    ####################################################################
+    # This is just an example based on the file snippet shown below.  It
+    # will vary depending on how your data looks.
+    ####################################################################
+    file_columns      => ['SM', 'ID', 'is_paired', 'filename', 'is_mate_1', 'read_length', 'is_13plus', 'CN', 'PL', 'DS'],
+    long_read_columns => ['sample','filename'],
+
+    # RNA-seq pipeline stuff
+    'rnaseq_dir'    => catdir($self->o('output_path'), 'rnaseq'),
```

Comment:
Could you please remove this block as none of the values are needed in this pipeline?

Reply:
Deleted all but the last variable, which has references to it in the pipeline:
```
    'rnaseq_dir'    => catdir($self->o('output_path'), 'rnaseq'),
```